### PR TITLE
Support for multiple schemas/projects synchronization

### LINKED ISSUE #66: One docker container for multiple syncs
Currently, you need to run one docker per schema to sync between PG and Mergin Maps. It would be good to have an option to support multiple schemas and projects running with one docker container.

### PR DESCRIPTION
**Pull request by alexbruy**
Adds support for multiple schemas/projects. As INI format does not support nested sections, config file format was changed to YAML.

Fixes #66.

Funded by [County Broadband](https://countybroadband.co.uk/).

ref CU-2h19n1g

### CODE REVIEW

File: dbsync.py [hunk 1]
```diff
@@ -287,396 +205,431 @@ def create_mergin_client():
 def dbsync_pull(mc):
     """ Downloads any changes from Mergin Maps and applies them to the database """
 
-    _check_has_working_dir()
-    _check_has_sync_file()
+    for sch in config.schemas:
+        schema_config = [sch.driver, sch.conn_info, sch.modified, sch.base, sch.mergin_project, sch.sync_file]
+        if not all(schema_config):
```
Comment (varmar05) [inline dbsync.py:210]: don't we have validation elsewhere?

**Review by varmar05 — APPROVED** [review]

**Comment by varmar05** [comment]
ref CU-2h19n1g

File: Dockerfile [hunk 2]
```diff
@@ -17,7 +17,8 @@ RUN apt-get update && apt-get install -y \
 
 # Python Mergin client
 RUN python3 -m pip install --upgrade pip
-RUN pip3 install mergin-client==0.7.3 dynaconf==3.1
+RUN pip3 install mergin-client==0.7.3
```
Comment (varmar05) [inline Dockerfile:20]: shall we start writing requirements file? it will be easier also to set up development

Comment (alexbruy) [reply Dockerfile:20]: I think we should go even further and unify approaches used in media-/dbsync and probably python client. E.g. use pipenv everywhere.

Comment (varmar05) [reply Dockerfile:20]: sounds good to me

**Pull request merged**